force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +455,15 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        }
+        if (person1.getAge() < person2.getAge()) {
+            return person2;
+        }
+        else {
+            return person2;
+        }
```

Comment:
You can simplify this if you want. And have just one if condition. If the values are a tie, you can return either.